packages/frontend/src/app/invite/page.tsx

```diff
@@ -0,0 +1,17 @@
+import { NavigationBreadcrumbs } from '@/common/components/navigation/Breadcrumbs'
+import { getAuthToken } from '@/common/helpers'
+import { InviteForm } from './components/InviteForm'
+
+export default function InvitePage() {
+  const authToken = getAuthToken()
+
+  return (
+    <main className="flex flex-col w-full items-center">
+      <NavigationBreadcrumbs rootHref="/" rootName="Trusted Entities" />
```

Comment:
I wonder how much worse it would be just to use the following? 🤔 
```suggestion
      <Link href="/">Trusted Entities</Link> > Invite
```

Reply:
But what about the other pages? And this would only display Trusted Entities, not entire breadcrumbs.